rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1834,6 +1847,8 @@ protected void executeOpcode() {
             break;
             case OpCodes.OP_GASLIMIT: doGASLIMIT();
             break;
+            case OpCodes.OP_BASEFEE: doBASEFEE();
```

Comment:
probably we'd need to add here a check for activation height and throw an exception (`Program.ExceptionHelper.invalidOpCode()`), if it's not activated yet

Reply:
Yup!! Let's add that!